test/RelayedTransactionValidator.test.ts

```diff
@@ -0,0 +1,272 @@
+import { expect, use } from 'chai';
+import sinonChai from 'sinon-chai'
+import { createSandbox, SinonStubbedInstance } from 'sinon';
+import { RelayedTransactionValidator } from '../src/RelayedTransactionValidator';
+import ContractInteractor from '@rsksmart/rif-relay-common/dist/src/ContractInteractor';
+import chaiAsPromised from 'chai-as-promised';
+import type { DeployTransactionRequest, EnvelopingConfig } from '@rsksmart/rif-relay-common/dist/src';
+import { BigNumber, Transaction, Wallet } from 'ethers';
+
+use(sinonChai);
+use(chaiAsPromised);
+
+const sandbox = createSandbox();
+const createRandomAddress = () => Wallet.createRandom().address;
```

Comment:
We should have this in a utility for testing

Reply:
Will include in the next PR for sure